Documentation/workflow/UsingYourBuild.md

```diff
@@ -1,43 +1,51 @@
 
 # Using your .NET Core Runtime Build
 
-We assume that you have successfully built CoreCLR repository and thus have file of the form  
+We assume that you have successfully built CoreCLR repository and thus have file of the form
 ```
     bin\Product\<OS>.<arch>.<flavor>\.nuget\pkg\Microsoft.NETCore.Runtime.CoreCLR.<version>.nupkg
 ```
 And now you wish to try it out.  We will be using Windows OS as an example and thus will use \ rather
 than / for directory separators and things like Windows_NT instead of Linux but it should be
-pretty obvious how to adapt these instructions for other operating systems.  
+pretty obvious how to adapt these instructions for other operating systems.
 
 To run your newly built .NET Core Runtime in addition to the application itself, you will need
 a 'host' program that will load the Runtime as well as all the other .NET Core Framework
-code that your application needs.   The easiest way to get all this other stuff is to simply use the 
+code that your application needs.   The easiest way to get all this other stuff is to simply use the
 standard 'dotnet' host that installs with .NET Core SDK.
 
 The released version of 'dotnet' tool may not be compatible with the live CoreCLR repository. The following steps
-assume use of unreleased version of 'dotnet' tool that is downloaded as part of the CoreCLR repository
-build at `<repo root>\Tools\dotnetcli`. [Add `Tools\dotnetcli` directory to your path](../building/windows-instructions.md#adding-to-the-default-path-variable)
-and type:
+assume use aquire a dogfood build of the .NET SDK
 
-* dotnet -?
+## Install the latest nightly .NET Core SDK 2.0
 
-and it prints some help text, you are ready.
+- [Win 64-bit Latest](https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-dev-win-x64.latest.exe)
```

Comment:
Yeah, the zip files are probably a bit safer to use for simple dogfooding. I think you can just replace "exe" with "zip" and get a link from the above, e.g.: https://dotnetcli.blob.core.windows.net/dotnet/Sdk/master/dotnet-dev-win-x64.latest.zip